inference-engine/tests/functional/plugin/cpu/shared_tests_instances/extension/extension.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (C) 2020 Intel Corporation
+// Copyright (C) 2020-2021 Intel Corporation
```

Comment:
Why `shared_test_instances`??? I suppose to move this folder upder `functional/plugin/cpu` folder